oqs-template/ssh-oqs.c/define_sig_functions.fragment

```diff
@@ -112,5 +112,38 @@ const struct sshkey_impl sshkey_{{ symbol_base_name }}_impl = {
 };
 {%- endfor %}
 {%- endfor %}
+#ifdef OPENSSL_HAS_ECC
+{%- for sig in config['sigs'] %}
+{%- for alg in sig['mix_with'] if not alg['rsa'] %}
+{%- set symbol_base_name = alg['name']|replace('_','') + '_' + sig['name']|replace('_','') %}
+static const struct sshkey_impl_funcs sshkey_{{ symbol_base_name }}_funcs = {
+  /* .size = */ ssh_generic_size,
+  /* .alloc = */ ssh_generic_alloc,
+  /* .cleanup = */ ssh_generic_cleanup,
+  /* .equal = */ ssh_generic_equal,
+  /* .ssh_serialize_public = */ ssh_generic_serialize_public,
+  /* .ssh_deserialize_public = */ ssh_generic_deserialize_public,
+  /* .ssh_serialize_private = */ ssh_generic_serialize_private,
+  /* .ssh_deserialize_private = */ ssh_generic_deserialize_private,
+  /* .generate = */ ssh_generic_generate,
+  /* .copy_public = */ ssh_generic_copy_public,
+  /* .sign = */ ssh_generic_sign,
+  /* .verify = */ ssh_generic_verify,
+};
+
+const struct sshkey_impl sshkey_{{ symbol_base_name }}_impl = {
+  /* .name = */ "ssh-{{ alg['name']|replace('_','-') + '-' + sig['name']|replace('_','') }}",
+  /* .shortname = */ "{{ alg['name']|upper + '_' + sig['name']|replace('_','')|upper }}",
+  /* .sigalg = */ NULL,
+  /* .type = */ KEY_{{ alg['name']|upper }}_{{ sig['name']|upper }},
+  /* .nid = */ {{ alg['openssl_nid'] }},
+  /* .cert = */ 0,
+  /* .sigonly = */ 0,
+  /* .keybits = */ 256, // TODO - What should be here?
```

Comment:
Doesn't look quite finished, does it?

Reply:
It's used as a fallback if `impl->size()` is NULL [here](https://github.com/open-quantum-safe/openssh/blob/OQS-v9/sshkey.c#L451). For OQS, this is never the case so keybits should be set to 0 (like it is for ECDSA and RSA).